Sexuality/Homosexuality.md

```diff
@@ -1,26 +1,26 @@
 #Homosexuality
 
 ##Gender dynamism
-I think we need to really start to listen to the Tumblr girls.
-Sexual orientation and gender can be dynamic.
+I think we need to start to listen to the Tumblr girls. 
+Sexual orientation and gender can be dynamic
 
-##Occourrence of homosexuality in other kingdom
-Homosexual behavior exist in nature because it brings an advantage to the population. (source missing)
-Could be both learned or genetic. There are yet to be genetically analysis that demonstrate the genetic linkage.
-Other animals have homosexual behavior, we lack records from the past below the 70ies because biologist,
-being humans, ignored such behaviors.
-Lions use it for dominance assessing (same as the Roman empire), penguins being semelpair (like most of the birds)
-engage homosexual behavior life long.
+##Occurrences of homosexuality in other kingdoms
+Homosexual behavior exists in nature because it brings an advantage to the population. (source missing).
+This behavior can be either learned or genetic. A genetic analysis that demonstrate the genetic linkage has yet to be discovered. 
+Other animals have expressed homosexual behavior, but we lack records prior to 1970 because biologists 
+ignored such behaviors. 
+Lions use it for dominance assessing (same as the Roman empire), penguins being semelparous (like most birds) 
+engage homosexual behavior their entire life. 
 
 ##Personal choice or predisposition
-Homosexuality isn't a choice, like isn't a choice to be hetero. The sexual attraction, albeit can be guided, cannot be completely replaced. I would call it a predisposition. 
-Sexual orientation and sexual preference come both from our conscious and subconscious thoughts.
-There are gay reformatory, were people try to impose the heterosexuality. I have no idea about the success rate,
+Homosexuality isn't a choice, like it isn't a choice to be hetero. The sexual attraction, although it can be guided, it cannot be completely replaced. I would consider it a predisposition. 
```

Comment:
Call in the sense I would define it a predisposition. Considering is much more stronger...